README.md

```diff
@@ -201,3 +201,58 @@ AWS OpsWorks Chef Deployment
   ```ruby
   include_recipe 'datadog::dd-agent'
   ```
+
+Resources
+=========
+
+datadog_monitor
+---------------
+
+The `datadog_monitor` resource will help you to enable Agent integrations.
+
+The default action `:add` of this resource will automatically enable the integration by filling values in the configuration template file and set the correct permissions after having written the configuration on the filesystem. Also, it will take care of restarting the Agent.
+
+The `:remove` action disables an integration from a node.
+
+
+### Syntax
+
+```ruby
+datadog_monitor 'name' do
+  init_config                       Hash, default: {}
+  instances                         Array, default: []
+  logs                              Array, default: []
+  use_integration_template          true, false, default: false
+  action                            Symbol, default: :add
+end
+```
+
+#### Actions
+
+There is two available actions in this resource:
+
+* `:add` Default. Enable the integration.
+* `:remove` Use this action to disable the integration.
+
+#### Properties
+
+* `'name'` is the name of the Agent integration to configure and enable
+* `instances` are the fields used to fill values under the `instances` section in the integration configuration file.
+* `init_config` are the fields used to fill values under the the `init_config` section in the integration configuration file.
+* `logs` to configure the logs section in the integration configuration file.
```

Comment:
I think you should use the same sentence structure to describe `logs` as you did for `instances` and `init_config` (it would reflect that it should be used in the same way)